### DISCUSSION TITLE
Return list based on date range

### DISCUSSION THREAD

**Discussion by cmaclennan** [Budibase/budibase#9085] [Help]
Hi All, 

Hoping someone could help me or point me in the right direction, I want to return a list of rooms that are currently occupied based on check in/check out dates on booking table. 

Any insight is greatly appreciated

Thanks

**Comment by melohagan** [comment]
Hey @cmaclennan 

Could you not use the **Is empty** filter on the check-out date? If there is no check-out date then the room is still occupied?
